bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/InsteonBinding.java

```diff
@@ -155,34 +161,12 @@ public InsteonBinding(InsteonNetworkHandler handler, @Nullable InsteonNetworkCon
 
         deadDeviceTimeout = devicePollIntervalMilliseconds * DEAD_DEVICE_COUNT;
         logger.debug("dead device timeout set to {} seconds", deadDeviceTimeout / 1000);
-
-        String port = config.getPort();
-        logger.debug("port = '{}'", Utils.redactPassword(port));
-        driver.addPort("port", port, serialPortManager);
-        driver.addMsgListener(portListener, port);
-
-        logger.debug("setting driver listener");
-        driver.setDriverListener(portListener);
     }
```

Comment:
This is where I would suggest to move the code.
```suggestion
        String port = config.getPort();
        logger.debug("port = '{}'", Utils.redactPassword(port));

        driver = new Driver(port, portListener, serialPortManager);
        driver.addMsgListener(portListener);
    }
```

Reply:
If you look there is code above of this that depends on driver being initialized.

Reply:
Ah, didn't notice that. My bad.